simple-operations.py

```diff
@@ -0,0 +1,10 @@
+a=5
+b=10
+sum=a+b
+sub=a-b
+devide=a/b
+multiply=a*b
+print("sum of a and b is",sum)
+print("sub of a and b is",sub)
+print("devide of a and b is",devide)
+print("multiply of a and b is",multiply)
```

Comment:
multiply of a & b, is this correct English?

Reply:
Where are the screenshots?

Reply:
Screenshot done

Reply:
english corrected